_starter-kit/_patterns/_utility-classes.scss

```diff
@@ -1,37 +1,89 @@
+// Spacing directions
```

Comment:
Shouldn't this top bit be in tools somewhere?

Reply:
Yes and no.